### ISSUE TITLE
Chrome "Launch as Window" Apps won't open when HammerSpoon is turned on 

### ISSUE THREAD

**Issue by GraphicThinkPad** [jasonrudolph/keyboard#69]
This might be better to put on HammerSpoon's repo, but I thought I'd try here first. If you use Chrome browser, you can go to Settings > More Tools > Create Shortcut and tick the "Open as Window" option to make a pseudo web app. Oddly enough, once these shortcuts are activated (specifically the HammerSpoon ones), these windows simply won't open no matter how they're opened. I attached a gif of myself trying to open my Todoist app (which I made as such a shortcut) to show what I mean. I also kept the HammerSpoon console open in case that provides any clues.

I'd love any help with this, but since I don't know if this is a keyboard issue or a HammerSpoon issue, I'll keep my hopes low.

Either way, thank you!

![Keyboard error](https://user-images.githubusercontent.com/26067990/86419821-7ad93300-bca2-11ea-9cb1-5a752dbda977.gif)

**Comment by jasonrudolph** [comment]
👋 @GraphicThinkPad: Thanks for reporting this behavior. I tried to reproduce it, but I haven't been able to do so. Here's the behavior I'm seeing with macOS 10.15.5, Chrome 83.0.4103.116, and Hammerspoon 0.9.78:

![demo](https://user-images.githubusercontent.com/2988/86492056-c8a17a00-bd3a-11ea-8952-217ea488aca5.gif)

If I should use different steps to recreate the issue, please let me know.

I don't _think_ the problem you're seeing is specific to this repository, so I'm going to close this out. If you do discover that there's something about the specific Hammerspoon config in this repository that's causing the issue, please let me know. :pray:

I hope you're able to figure it out. Chrome Apps are pretty cool, and it would be a bummer to not be able to use them.

**Comment by GraphicThinkPad** [comment]
You know what? I made a silly mistake when creating this issue: I neglected to mention that the bug appeared when using Brave Browser, which has Chromium at its core. I tried to make an app with the Chrome browser itself and it worked just fine. It's a bit annoying to have to use a different browser for these full-window webpages, but it's perfectly workable.

I appreciate you checking into this, and thanks for the great config.
